src/app/components/Switch/index.tsx

```diff
@@ -9,20 +9,12 @@ interface Props {
 
 const Switch = ({ label }: Props) => (
   <form>
```

Comment:
o switch não pode estar dentro do form. Quem for implementar ele que precisa se preocupar com isso.